README.md

```diff
@@ -15,12 +15,17 @@ See [Architecture Docs](./docs/Architecture.md) for more details.
 ### Quick Start
 
 ```
+# Target genesis url for bdjuno to parse from
+export GENESIS_URL=https://raw.githubusercontent.com/likecoin/mainnet/master/genesis.json
+
 make setup
 docker-compose up
 
 # In another tab
 
 make -C react-app dev
+
+Note that it will take BDJuno some time to get data populated to the database
```

Comment:
best if we provide a snapshot db dump in the future?

Reply:
i think can, but can't help those who want to use LikeDAO with other chains

Reply:
ok ah